src/Api/Controllers/AnnotationController.Layers.cs

```diff
@@ -77,7 +77,7 @@ public IActionResult PostLayerRelationRule([FromBody] RelationFormModel model)
         [ProducesResponseType(typeof(void), 200)]
         [ProducesResponseType(typeof(void), 403)]
         [ProducesResponseType(typeof(void), 400)]
-        public IActionResult PutLayerRelationRule([FromBody] RelationFormModel original, [FromBody] RelationFormModel changed)
+        public IActionResult PutLayerRelationRule([FromBody] RelationFormModel original, [FromQuery] RelationFormModel changed)
```

Comment:
This changes the public REST interface of the API... Have you tested whether the API still behaves correctly after this change?